source/_integrations/event.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: Event
+description: Instructions on how to use event entities in Home Assistant.
+ha_category:
+  - Events
+ha_release: 2023.8
+ha_quality_scale: internal
+ha_domain: event
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+Events are signals that are emitted when something happens, for example, when a user presses a physical button like a doorbell or when a button on a remote control is pressed.
+
+These events are stateless. For example, a doorbell does not have a state like being "on" or "off" but instead is momentarily pressed. Some events can have variations in the type of event that is emitted. For example, maybe your remote control is capable of emitting a single press, a double press, or a long press.
+
+The event entity can capture these events in the physical world and makes them available in Home Assistant as an entity.
+
+Please note the event integration cannot be directly used; you cannot create your own event entities using this integration. This integration is a building block for other integrations to use, enbaling them to create event entities for you.
```

Comment:
```suggestion
Please note the event integration cannot be directly used; you cannot create your own event entities using this integration. This integration is a building block for other integrations to use, enabling them to create event entities for you.
```

Reply:
Thanks for pointing that out! I need to add another update as well, I will fix that in the next go.